components/claim/ClaimList.tsx

```diff
@@ -60,7 +60,7 @@ export const ClaimList = (props: Props) => {
   // that search queries are automatically extended at the moment in order to
   // provide claims with comments and comments with their parent claims. This
   // automatic extension is the reason for our filtering efforts here.
-  const list = getLis(claims.data, votes.data, props.page || "");
+  const list = getLis(claims.data || [], votes.data || [], props.page || "");
```

Comment:
Ignore the changes here. I fixed a bug the other day and hadn't committed it. Tis what it is.